packages/circuit-ui/components/Table/types.ts

```diff
@@ -25,7 +25,7 @@ export type CellObject = {
   sortByValue?: SortByValue;
 };
 
-export type Cell = string | number | CellObject;
+export type Cell = string | number | CellObject | null | undefined;
```

Comment:
Actually there's probably a good case for `undefined`, especially since Tables are usually populated with data from API responses, where undefined is plausible

Reply:
Right, that was my logic as well, to support both common void values.